MeshLib/MeshGenerators/MeshLayerMapper.cpp

```diff
@@ -60,6 +61,10 @@ MeshLib::Mesh* MeshLayerMapper::createStaticLayers(MeshLib::Mesh const& mesh, st
 	std::vector<MeshLib::Node*> new_nodes(nNodes + (nLayers * nNodes));
 	std::vector<MeshLib::Element*> new_elems;
 	new_elems.reserve(nElems * nLayers);
+	MeshLib::Properties properties;
+	boost::optional<PropertyVector<int> &> materials = 
+		properties.createNewPropertyVector<int>("MaterialIDs", MeshLib::MeshItemType::Cell);
+	materials->reserve(nElems * nLayers);
```

Comment:
The ```materials``` variable is definitely empty at that point because it's created just one line above. It's *supposed* to be empty ... I am just creating the mesh...